units/en/unit1/additional-readings.mdx

```diff
@@ -0,0 +1,11 @@
+# Additional Readings [[additional-readings]]
+
```

Comment:
Maybe add a sentence explaining that these are optional resources to go in more depth

Reply:
:heavy_check_mark: